update.sh

```diff
@@ -299,7 +302,7 @@ print_expose_ports() {
 	cat >> $1 <<-EOI
 	# $expose_comment
 	EXPOSE $expose_ports
-
+	
```

Comment:
This tab is not used so I think it can be removed.

Reply:
But the Dockerfile looks nicely formatted.